### ISSUE TITLE
run comparison

### ISSUE THREAD

**Issue by eckshton** [absolllute/Mega-Hack-Pro-Future#1115]
it'd be cool if you could compare your run against a completion and see how many frames off each of your inputs is. maybe you could select a botted run to compare against or something, idk. would be cool.

**Comment by CoderQCFromScraatch** [comment]
The reason Show Trajectory takes away performance I think is something called Trignometry. They take many more clock cycles than normal math but allow for stuff like movement based on direction. Combined with the checks it does and rendering, this takes a lot of power.

**Comment by Brittank88** [comment]
> The reason Show Trajectory takes away performance I think is something called Trignometry. They take many more clock cycles than normal math but allow for stuff like movement based on direction. Combined with the checks it does and rendering, this takes a lot of power.

It's less about specifically the use of trigonometric functions, and more about having to run a simulation of where the player goes from the point that you're actually at. This involves running all the physics calculations I'm pretty sure.

**Comment by Brittank88** [comment]
In terms of OP's idea, I think this is pretty cool. Deviance statistics between runs would be nice.

**Comment by NewAlexGamer** [comment]
less accurate, less resource heavy

**Comment by eckshton** [comment]
yeah i dont know anything about show trajectory thats why i was thinking a brute force method like logging clicks could work instead. maybe even have both available but swap between the two
